drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/custom_assessment_loader.rb

```diff
@@ -114,17 +118,36 @@ def create_assessment_records(rows)
     end
 
     # create synthetic form processors for imported touchpoints
+    # also link form processors to CE assessments if ce_assessment_level is present
     def create_form_processor_records(rows)
+      ce_assessments_by_date_and_enrollment_id = {}
+      if ce_assessment_level
+        ce_assessments_by_date_and_enrollment_id = Hmis::Hud::Assessment.
```

Comment:
This should be adjusted so it only looks in the HMIS data source. Otherwise we could find matches with CE Assessments in other data sources.
```suggestion
        ce_assessments_by_date_and_enrollment_id = Hmis::Hud::Assessment.hmis.
```